### DISCUSSION TITLE
contributing

### DISCUSSION THREAD

**Discussion by JackyBorderCollie** [CluckCluckChicken/Magnifier#26] [General]
aww come on i want to contribute too

**Comment by davwat** [comment]
find the imposter then make pull request to deactivate the sus

**Reply by JackyBorderCollie** [reply]
waaaaaah ok

**Reply by JackyBorderCollie** [reply]
also how do i find sus

**Reply by davwat** [reply]
clean the electrical sus vents

**Comment by JackyBorderCollie** [comment]
yess i'm one of the contributors

**Comment by JackyBorderCollie** [comment]
ok pls close this
